src/components/TokenBalanceCard/TokenBalanceCardTemplate.tsx

```diff
@@ -17,45 +17,49 @@ type Props = {
   | "withSymbol"
   | "tokenSymbol"
   | "precision"
-  | "withHigherPrecision"
   | "higherPrecision"
+  | "isLarge"
 >
 
 const TokenBalanceCardTemplate: FC<Props> = ({
-  icon,
   title,
+  tokenSymbol,
   tokenBalance,
   usdBalance,
   contract,
-  tokenSymbol,
   tokenDecimals,
   tokenFormat,
-  withHigherPrecision,
   precision,
   higherPrecision,
   withSymbol = false,
+  isLarge = false,
   ...restProps
 }) => {
   return (
-    <Card {...restProps} as="section">
-      <HStack as="header" mb="4">
-        <Icon boxSize="16px" as={icon} />
-        <LabelSm>{title}</LabelSm>
-      </HStack>
+    <VStack spacing={isLarge ? 4 : 1} alignItems="flex-start" {...restProps}>
+      <Text
+        as="h2"
+        fontSize={isLarge ? "lg" : "md"}
+        lineHeight={6}
+        color="hsl(0, 0%, 50%)"
+        fontWeight="medium"
+      >
+        {title}
+      </Text>
       <TokenBalance
         tokenAmount={tokenBalance}
-        usdBalance={usdBalance}
         tokenSymbol={tokenSymbol}
-        withSymbol={withSymbol}
+        usdBalance={usdBalance}
         tokenDecimals={tokenDecimals}
         tokenFormat={tokenFormat}
-        withUSDBalance
-        withHigherPrecision={withHigherPrecision}
         precision={precision}
         higherPrecision={higherPrecision}
+        isEstimated
```

Comment:
I think `isEstimated` is not needed here. The `TokenBalance` component doesn't have `isEstimated` prop so this results in this warning in console:

```
Warning: React does not recognize the `isEstimated` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isestimated` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Either way, `isEstimated` property should not be set to true in case to tbtc value as it is real value, not estimated. The only case where it will use the `~` mark should be when the precision (or higher precision in case of tooltip) is smaller than the real value. I think it does not work as expected right now (it only works if the value is smaller than 1) but this is something that we should fix in a future PR.

For now, let's just remove that `isEstimated` prop that is being passed to `TokenBalance`

Reply:
Ref commit: 90c7608b